app/javascript/mastodon/features/ui/components/navigation_panel.js

```diff
@@ -79,7 +80,7 @@ class NavigationPanel extends React.Component {
             <ColumnLink transparent to='/bookmarks' icon='bookmark' text={intl.formatMessage(messages.bookmarks)} />
             <ColumnLink transparent to='/lists' icon='list-ul' text={intl.formatMessage(messages.lists)} />
 
-            <ListPanel />
+            {!isMobile(window.innerWidth) && <ListPanel />}
```

Comment:
I think it would be safer to hide this with CSS because this function will not take window resizing into account properly.

Reply:
fix https://github.com/mastodon/mastodon/compare/e95929054daa18f69e17855cef43352598211bf2..6c999f8ed4f0f3eb242970028df9d0e374728b5e